pkgs/top-level/ocaml-packages.nix

```
@@ -314,6 +314,8 @@ let
       then callPackage ../development/ocaml-modules/ipaddr { }
       else ipaddr_p4;
 
+    irmin = callPackage ../development/ocaml-modules/irmin { };
```

```suggestion
    irmin_1 = callPackage ../development/ocaml-modules/irmin/1.4.nix { };
```

Can you please leave room for irmin 2? Thanks.